SetupBLT.cmake

```diff
@@ -1,44 +1,7 @@
-###############################################################################
-# Copyright (c) 2017-2019, Lawrence Livermore National Security, LLC.
-#
-# Produced at the Lawrence Livermore National Laboratory
-#
-# LLNL-CODE-725085
```

Comment:
@white238 and others -- could you please confirm that we no longer need to include ``LLNL-CODE-725085``?

Should it go in ``COPYRIGHT`` or ``README.md``?

Reply:
It’s in the top level one.

Reply:
Thanks @white238 -- I think I missed it because it wasn't modified in this PR.